Assets/Mirror/Migration/Editor/Scripts.cs

```diff
@@ -35,11 +47,26 @@ public class Scripts : MonoBehaviour {
                 "uint",
                 "syncInterval",
                 "NetworkServer.connections.Values",
-                "NetworkClient"
+                "NetworkServer.connections.Values",
+                "NetworkClient",
+                "Mirror.NetworkAnimator",
+                "Mirror.NetworkIdentity",
+                "Mirror.NetworkManager",
+                "Mirror.NetworkProximityChecker",
+                "Mirror.NetworkStartPosition",
+                "Mirror.NetworkTransform",
+                "Mirror.NetworkTransformChild",
+                "Mirror.NetworkLobbyManager",
+                "Mirror.NetworkLobbyPlayer",
+                "Mirror.NetworkManagerHUD",
+                "((int)(NetworkTime.rtt*1000d))"
```

Comment:
Since NetworkClient.GetRTT() returns float, cast shouldn't be float instead of int?

Reply:
In Unet it returns ms in an int, in Mirror seconds in a float